### PR TITLE
fix: Make NPM package smaller by excluding demo

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     },
     "publishConfig": {
         "registry": "https://npm.pkg.github.com"
-    }
-    
+    },
+    "files": [
+        "assets/",
+        "README.md"
```
Comment (ddeboer) [inline package.json:16]: Any other files that need to be in the NPM package?

**Comment by commonpike** [comment]
having the demo in the package was on purpose. as the README suggests, you can download the zip and click index.html. 
and then check the source and never read the docs ;-) which is also why the demo is built and committed to the repo by the developer. 

we could also create a smaller package without the demo, and a separate package for the demo ?

the use case of including the sass files yourself is actually a bit of an outlier, but it should be possible.

**Comment by ddeboer** [comment]
> as the README suggests, you can download the zip 

That zip is not the NPM package but just the zipped source code, so demo still doesn’t need to be in the NPM package.

**Comment by commonpike** [comment]
> > as the README suggests, you can download the zip
> 
> That zip is not the NPM package but just the zipped source code, so demo still doesn’t need to be in the NPM package.

Oh! true. 

Then, yes, all we need is the /assets folder and the README.

**Pull request merged**